app/components/admin_index_component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AdminIndexComponent < ViewComponent::Base
+  def initialize(additional_links:, default: [])
```

Comment:
in answer to your q i think all you need to do is pass in all the params here :)

Reply:
OH sorry i read your comment again

this is making additional_links mandatory i think and it isnt? i am a bit rusty but i think `additional_links: [], default: []` should fix it ?